### ISSUE TITLE
Difficult to download

### ISSUE THREAD

**Issue by flip111** [mi-g/vdhcoapp#27]
I tried 5 times to download the Linux - 64 bits - deb. The download starts but does not complete. Transfer speed is also low.

**Comment by mi-g** [comment]
Are you downloading from this link [https://github.com/mi-g/vdhcoapp/releases/download/v1.1.3/net.downloadhelper.coapp-1.1.3-1_amd64.deb](https://github.com/mi-g/vdhcoapp/releases/download/v1.1.3/net.downloadhelper.coapp-1.1.3-1_amd64.deb)

**Comment by flip111** [comment]
Yes the link works, it's just not the best hosting when the connection is poor.

**Comment by mi-g** [comment]
Github is supposed to be very reliable. If you have internet connection problems, i'm not sure what other hosting solution we could use to make things better.
